File: redwood-yoga-java/src/main/java/io/github/orioncraftmc/meditate/internal/enums/YGExperimentalFeature.java
package io.github.orioncraftmc.meditate.internal.enums;

public enum YGExperimentalFeature {
    YGExperimentalFeatureWebFlexBasis;

    public static final int SIZE = java.lang.Integer.SIZE;

    public static YGExperimentalFeature forValue(int value) {
        return values()[value];
    }

    public int getValue() {
        return this.ordinal();
    }
}

File: redwood-yoga/src/commonMain/kotlin/app/cash/redwood/yoga/internal/enums/YGExperimentalFeature.kt
package app.cash.redwood.yoga.internal.enums

enum class YGExperimentalFeature {
  YGExperimentalFeatureWebFlexBasis;

  fun getValue(): Int {
    return ordinal
  }

  companion object {
    fun forValue(value: Int): YGExperimentalFeature {
      return values()[value]
    }
  }
}
